_schemaV2.graphql

```diff
@@ -8622,18 +8622,15 @@ type CreateVerifiedRepresentativeSuccess {
   verifiedRepresentative: VerifiedRepresentative
 }
 
-# Autogenerated input type of CreateViewingRoom
 input CreateViewingRoomInput {
   attributes: ViewingRoomAttributes
 
   # Main text
   body: String
-
-  # A unique identifier for the client performing the mutation.
   clientMutationId: String
 
   # End datetime
-  endAt: ISO8601DateTime
```

Comment:
Should I add `GravityISO8601DateTime` type? Similar to `GravityARImage` below

Reply:
nah, clients probably are fine either way since it's just a string. These fields may not even be used directly by clients vs. the other time fields like `distanceToClose`.